### PR TITLE
Integration test for --lvname, --lvsize and --rootfs flags

### PR DESCRIPTION
**Pull request by shishir-a412ed**
Signed-off-by: Shishir Mahajan <shishir.mahajan@redhat.com>

### CODE REVIEW

**Comment by shishir-a412ed** [comment]
`test_storage.sh` is passing on my local VM. Is there someway I can get access to the failed logs (tests.log) ?

ping @baude @jlebon

**Comment by jlebon** [comment]
The `tests.log` are available in the `artifacts/` directory when you click on the Details links down there. ⬇
It seems to be failing because `findmnt` in the cleanup is not finding anything. Note that you should also be able to reproduce the test setup using Vagrant.

**Comment by shishir-a412ed** [comment]
@jlebon yeah I found it right after posting the comment :) .. That's why I deleted the comment.

Thanks again !
Shishir

**Comment by shishir-a412ed** [comment]
ping @rhatdan

File: tests/integration/test_storage.sh [hunk 1]
```diff
@@ -116,4 +128,40 @@ EOF
...
+
+    ${ATOMIC} storage modify --lvname "$lvname" --lvsize "20%FREE" --rootfs "$rootfs"
+    grep -q "^CONTAINER_ROOT_LV_NAME=\"$lvname\"$" /etc/sysconfig/docker-storage-setup
+    grep -q "^CONTAINER_ROOT_LV_MOUNT_PATH=\"$rootfs\"$" /etc/sysconfig/docker-storage-setup
+    grep -q "^CONTAINER_ROOT_LV_SIZE=\"20%FREE\"$" /etc/sysconfig/docker-storage-setup
+
+    # atomic storage modify --lvname should throw error if passed without --rootfs.
+
+    set +e
+    OUTPUT=$(${ATOMIC} storage modify --lvname "$lvname" 2>&1)
```
Comment (rhatdan) [inline tests/integration/test_storage.sh]: Should we change the name and make sure the change works?

File: tests/integration/test_storage.sh [hunk 2]
```diff
@@ -116,4 +128,40 @@ EOF
...
+    if [[ $? -eq 0 ]]; then
+        exit 1
+    fi
+    set -e
```
Comment (rhatdan) [inline tests/integration/test_storage.sh]: No need for this set +e?  Shouldn't we just run set +e for the entire script?

**Comment by rhatdan** [comment]
Give some bad data for the lvsize and make sure it fails.

**Comment by shishir-a412ed** [comment]
@rhatdan I have added a test for passing bad data to lvsize, and check if it fails.

**Comment by rhatdan** [comment]
You did not push.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+ 25a4891

**Pull request closed without merging**